docgen/src/examples/e-commerce-infinite/App.js

```diff
@@ -160,7 +120,8 @@ const ColorItem = ({item, createURL, refine}) => {
 };
 
 const CustomColorRefinementList = ({items, refine, createURL}) =>
-    <div>
+    items.length > 0 ? <div>
+      <h5 className={'ais-Panel__Title'}>Colors</h5>
```

Comment:
Did you already thought about how to allow creating your own panels? This could answer this use case in a pretty way. Basically we need to be able to have a connectPanel method that would provide an "isEmpty" or "isRelevant" or "canRefine" boolean to indicate that some widget or custom widget inside has ways to refine.